src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -343,6 +343,54 @@ public void testNoGTs() throws IOException {
         spec.executeTest("testNoGTs--" + testFile, this);
     }
 
+    @Test
+    public void testRemoveMonomorphAfterSNSelect() throws IOException {
+        final String testFile = getToolTestDataDir() + "spanning_deletion.vcf";
+        final String samplesFile = "NA1";
```

Comment:
Really this is a sample name rather than a samplesFile so can you change the variable name?